Patch-Source: https://github.com/latchset/luksmeta/pull/10

From af81262c0f6e6b37e5c6b520d0fca3a9f9b6c1fc Mon Sep 17 00:00:00 2001
From: Christoph Biedl <debian.axhn@manchmal.in-ulm.de>
Date: Sat, 28 Sep 2019 13:24:11 +0200
Subject: [PATCH] Force creation of LUKS1 containers in test suite

Cryptsetup defaults to LUKS2 since version 2.1, make sure to create
LUKS1 containers instead by using the "--type luks1" argument.
---
 test-luksmeta | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test-luksmeta b/test-luksmeta
index fd17ead..f1e8b2e 100755
--- a/test-luksmeta
+++ b/test-luksmeta
@@ -11,7 +11,7 @@ function onexit() {
 trap 'onexit' EXIT
 
 truncate -s 4M $tmp
-echo -n foo | cryptsetup luksFormat $tmp -
+echo -n foo | cryptsetup luksFormat --type luks1 $tmp -
 
 ! ./luksmeta test -d $tmp
 
